editor.planx.uk/src/pages/FlowEditor/components/Team/types.ts

```diff
@@ -15,15 +16,17 @@ export interface AddNewEditorModalProps {
   setShowModal: React.Dispatch<SetStateAction<boolean>>;
 }
 export interface AddNewEditorFormValues {
-  email: string;
+  email: string | null;
   firstName: string;
   lastName: string;
 }
```

Comment:
I understand the intention, but this seems like the wrong approach - in order to add a user, we do need an email. It seems incorrect that and interface called `AddNewEditorFormValues` could have a `null` email address.